PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -55,3 +55,27 @@ def miniAOD_customizeMETFiltersFastSim(process):
 phase2_hgcal.toReplaceWith( Flag_HBHENoiseIsoFilter, cms.Path() )
 phase2_hgcal.toReplaceWith( Flag_eeBadScFilter, cms.Path() )
 
+metFilterPathsTask = cms.Task(
+    HBHENoiseFilterResultProducer,
+    HBHENoiseFilter,
```

Comment:
Are the filters in metFilterPaths_cff.py ever meant to run unscheduled or are they always supposed to be actual filters on a Path? At least the one filter I checked actually produced a bool so it is possible they are intended to run unscheduled in some use cases, but if they are not then I could just delete this Task entirely.